Composer/packages/client/src/components/ManageLuis/ManageLuis.tsx

```diff
@@ -0,0 +1,680 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
this is huge, it can be broken down into multiple components, one for each step and one to handle the logic for showing them and managing the data.

Reply:
Agreed. It looks like ManageLuis and ManageQnA share a lot in common, so they could be refactored somehow.

Reply:
I agree and I can commit to refactoring these guys a bit into smaller and potentially shared components before the release, but I'd love to merge this before doing that since I'd like to get this into stakeholder hands ASAP.

WE COOL WITH THAT?